virtual-hyperscript/svg.js

```diff
@@ -42,6 +44,11 @@ function svg(tagName, properties, children) {
             continue;
         }
 
+        if (key.substr(0, 6) === 'xlink:') {
```

Comment:
I think we might be able to make this more efficient by putting the namespace information in the attribute lookup table